examples/bot-ready-signalling/client/react-native/.nvmrc

```diff
@@ -0,0 +1 @@
+18.18
```

Comment:
Should we use the current active lts node version (22)? Or at least the maintenance lts version (20)? 18 is on its last legs